src/main/java/emu/grasscutter/server/packet/recv/HandlerClientLockGameTimeNotify.java

```diff
@@ -11,6 +12,6 @@ public final class HandlerClientLockGameTimeNotify extends PacketHandler {
     @Override
     public void handle(GameSession session, byte[] header, byte[] payload) throws Exception {
         var packet = ClientLockGameTimeNotify.parseFrom(payload);
-        session.getPlayer().getWorld().lockTime(packet.getIsLock());
+        session.getPlayer().setProperty(PlayerProperty.PROP_IS_GAME_TIME_LOCKED, packet.getIsLock() ? 1 : 0);
```

Comment:
this should be able to take a boolean, correct me if im wrong

Reply:
Maybe, but I wanted to play it safe.